Property '[Symbol.iterator]' is missing

Property '[Symbol.iterator]' is missing in every iterable class in `@minecraft/server/index.d.ts` file.

## tsc debug
```
node_modules/@minecraft/server/index.d.ts(2811,14): error TS2420: Class 'EnchantmentList' incorrectly implements interface 'Iterable<Enchantment>'.
  Property '[Symbol.iterator]' is missing in type 'EnchantmentList' but required in type 'Iterable<Enchantment>'.
node_modules/@minecraft/server/index.d.ts(4147,14): error TS2420: Class 'EntityIterator' incorrectly implements interface 'Iterable<Entity>'.
  Property '[Symbol.iterator]' is missing in type 'EntityIterator' but required in type 'Iterable<Entity>'.
node_modules/@minecraft/server/index.d.ts(5566,14): error TS2420: Class 'EntityTypeIterator' incorrectly implements interface 'Iterable<EntityType>'.
  Property '[Symbol.iterator]' is missing in type 'EntityTypeIterator' but required in type 'Iterable<EntityType>'.
node_modules/@minecraft/server/index.d.ts(6649,14): error TS2420: Class 'ItemTypeIterator' incorrectly implements interface 'Iterable<ItemType>'.
  Property '[Symbol.iterator]' is missing in type 'ItemTypeIterator' but required in type 'Iterable<ItemType>'.
node_modules/@minecraft/server/index.d.ts(13338,14): error TS2420: Class 'PlayerIterator' incorrectly implements interface 'Iterable<Player>'.
  Property '[Symbol.iterator]' is missing in type 'PlayerIterator' but required in type 'Iterable<Player>'.
```

--- a/.github/workflows/cl.yml
+++ b/.github/workflows/cl.yml
@@ -1,4 +1,4 @@
-name: gametest
+name: API
 
 on: [push, pull_request]
 

--- a/@minecraft/server-admin/README.md
+++ b/@minecraft/server-admin/README.md
@@ -8,3 +8,19 @@
 ### [store and load variables](./variables.md)
 
 ### [store and load secrets](./secrets.md)
+
+## Available Versions
+- 1.0.0-beta
+```json
+{
+    // @minecraft/server-admin
+    "uuid": "53d7f2bf-bf9c-49c4-ad1f-7c803d947920",
+    "version": "1.0.0-beta"
+}
+```
+```json
+{
+    "module_name": "@minecraft/server-admin",
+    "version": "1.0.0-beta"
+}
+```

--- a/@minecraft/server-gametest/README.md
+++ b/@minecraft/server-gametest/README.md
@@ -1,5 +1,5 @@
 # mojang-minecraft Module
 
-## Script examples
+## Module Contents
 
-### [Spawn simulated player](/scripts/spawn-simulated-player)
+### [Spawn simulated player](./spawn-simulated-player.md)

--- a/@minecraft/server-gametest/README.md
+++ b/@minecraft/server-gametest/README.md
@@ -3,3 +3,19 @@
 ## Module Contents
 
 ### [Spawn simulated player](./spawn-simulated-player.md)
+
+## Available Versions
+- 1.0.0-beta
+```json
+{
+    // @minecraft/server-gametest
+    "uuid": "6f4b6893-1bb6-42fd-b458-7fa3d0c89616",
+    "version": "1.0.0-beta"
+}
+```
+```json
+{
+    "module_name": "@minecraft/server-gametest",
+    "version": "1.0.0-beta"
+}
+```

--- a/@minecraft/server-gametest/README.md
+++ b/@minecraft/server-gametest/README.md
@@ -2,4 +2,4 @@
 
 ## Script examples
 
-### [Spawn simulated player](../scripts/spawn-simulated-player)
+### [Spawn simulated player](/scripts/spawn-simulated-player)

--- a/@minecraft/server-gametest/spawn-simulated-player.md
+++ b/@minecraft/server-gametest/spawn-simulated-player.md
@@ -1,0 +1,29 @@
+# Spawn simulated player
+
+You have to spawn a simulated player through a gametest test. To do this, register a gametest.
+
+```js
+import * as GameTest from '@minecraft/server-gametest';
+import { BlockLocation } from '@minecraft/server';
+
+// Name of the class of tests this test should be a part of.
+const testClassName = 'testClass';
+// Name of this specific test.
+const testName = 'test';
+// Implementation of the test function.
+/**
+ * @param {GameTest.Test} test
+ */
+const testFunction = (test) => {
+  test.spawnSimulatedPlayer(new BlockLocation(0, 1, 0)); // Spawns simulated player on 0, 1, 0 from test origin
+  // Simulated player code here
+};
+
+GameTest.register(testClassName, testName, testFunction)
+```
+
+To make simulated player stays on world until server close or behavior pack reloads, call function `RegistrationBuilder.maxTicks` and set the parameter to a very high number so it wouldn't despawn.
+
+```js
+GameTest.register(testClassName, testName, testFunction).maxTicks(0x7fffffff);
+```

--- a/@minecraft/server-net/README.md
+++ b/@minecraft/server-net/README.md
@@ -14,3 +14,19 @@
 ### [Performs a simple HTTP get request.](./http-get/)
 
 ### [Performs a HTTP request with HTTP headers.](./http-request/)
+
+## Available Versions
+- 1.0.0-beta
+```json
+{
+    // @minecraft/server-net
+    "uuid": "777b1798-13a6-401c-9cba-0cf17e31a81b",
+    "version": "1.0.0-beta"
+}
+```
+```json
+{
+    "module_name": "@minecraft/server-net",
+    "version": "1.0.0-beta"
+}
+```

--- a/@minecraft/server-ui/README.md
+++ b/@minecraft/server-ui/README.md
@@ -10,4 +10,4 @@
 
 ## Script examples
 
-### [Display another form after a form](../scripts/another-form)
+### [Display another form after a form](./form-after-form.md)

--- a/@minecraft/server-ui/README.md
+++ b/@minecraft/server-ui/README.md
@@ -11,3 +11,33 @@
 ## Script examples
 
 ### [Display another form after a form](./form-after-form.md)
+
+## Available Versions
+- 1.0.0-beta
+```json
+{
+    // @minecraft/server-ui
+    "uuid": "2bd50a27-ab5f-4f40-a596-3641627c635e",
+    "version": "1.0.0-beta"
+}
+```
+```json
+{
+    "module_name": "@minecraft/server-ui",
+    "version": "1.0.0-beta"
+}
+```
+- 0.1.0
+```json
+{
+    // mojang-minecraft-ui
+    "uuid": "2bd50a27-ab5f-4f40-a596-3641627c635e",
+    "version": [ 0, 1, 0 ]
+}
+```
+```json
+{
+    "module_name": "mojang-minecraft-ui",
+    "version": [ 0, 1, 0 ]
+}
+```

--- a/@minecraft/server-ui/form-after-form.md
+++ b/@minecraft/server-ui/form-after-form.md
@@ -1,0 +1,28 @@
+# Display another form after a form
+
+First, create your first server ui form with ModalForm, ActionForm, or MessageForm.
+
+```js
+// pseudocode
+new ModalFormData()
+```
+
+Show the form to the player with show function with a callback.
+
+```js
+// pseudocode
+let player: Player;
+new ModalFormData().show(player).then((response) => {
+    // callback code
+});
+```
+
+In the callback call another form, it can be any of the three forms.
+
+```js
+// pseudocode
+let player: Player;
+new ModalFormData().show(player).then((response) => {
+  new ActionFormData().show(player);
+});
+```

--- a/@minecraft/server/README.md
+++ b/@minecraft/server/README.md
@@ -19,3 +19,33 @@
 ### [Cancel Watchdog Terminate](./watchdog/beforeWatchdogTerminate.js)
 
 ![](https://media.discordapp.net/attachments/852517995308646410/1009680002209546240/unknown.png)
+
+## Available Versions
+- 1.0.0-beta
+```json
+{
+    // @minecraft/server
+    "uuid": "b26a4d4c-afdf-4690-88f8-931846312678",
+    "version": "1.0.0-beta"
+}
+```
+```json
+{
+    "module_name": "@minecraft/server",
+    "version": "1.0.0-beta"
+}
+```
+- 0.1.0
+```json
+{
+    // mojang-minecraft
+    "uuid": "b26a4d4c-afdf-4690-88f8-931846312678",
+    "version": [ 0, 1, 0 ]
+}
+```
+```json
+{
+    "module_name": "mojang-minecraft",
+    "version": [ 0, 1, 0 ]
+}
+```

--- a/package.json
+++ b/package.json
@@ -17,17 +17,17 @@
   "homepage": "https://github.com/JaylyDev/GametestDB#readme",
   "type": "module",
   "devDependencies": {
+    "@minecraft/server": "^1.0.0-beta.11940b24",
+    "@minecraft/server-admin": "1.0.0-beta.11940b24",
+    "@minecraft/server-gametest": "1.0.0-beta.11940b24",
+    "@minecraft/server-net": "1.0.0-beta.11940b24",
+    "@minecraft/server-ui": "1.0.0-beta.11940b24",
     "@types/mojang-gametest": "^0.1.6",
     "@types/mojang-minecraft": "^0.1.7",
     "@types/mojang-minecraft-server-admin": "^0.1.1",
     "@types/mojang-minecraft-ui": "^0.1.0",
     "@types/mojang-net": "^0.1.1",
     "@types/node": "^18.7.14",
-    "typescript": "^4.8.2",
-    "@minecraft/server": "1.0.0-beta.11940b23",
-    "@minecraft/server-ui": "1.0.0-beta.11940b23",
-    "@minecraft/server-net": "1.0.0-beta.11940b23",
-    "@minecraft/server-admin": "1.0.0-beta.11940b23",
-    "@minecraft/server-gametest": "1.0.0-beta.11940b23"
+    "typescript": "^4.8.2"
   }
 }

